.github/workflows/macos_meson.yml

```diff
@@ -34,9 +34,10 @@ jobs:
         submodules: recursive
 
     - name: Setup Python
-      uses: actions/setup-python@v3
+      uses: actions/setup-python@v4
```

Comment:
Is this `setup-python` needed at all? This job uses Miniconda/mamba.

I think you can drop it, and also remove the `python -m pip install meson==0.61.1` line (a newer `meson` version is already in the environment by now).

Reply:
Done, I also updated the mamba env setting as per the action's doc. It seems to be faster that way.